src/main/java/org/javawebstack/abstractdata/mapper/DefaultMappers.java

```diff
@@ -161,6 +153,24 @@ public Object fromAbstract(MapperContext context, AbstractElement element, Class
             }
         }
 
+        public Class<?>[] getSupportedTypes() {
```

Comment:
Maybe Stack and Vector should also be added?

Reply:
added